hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -215,6 +215,9 @@ public static void main(final String... args) throws Exception {
         final var merkleCryptography = MerkleCryptographyFactory.create(configuration, cryptography);
         MerkleCryptoFactory.set(merkleCryptography);
 
+        // Register with the ConstructableRegistry classes which need configuration.
+        BootstrapUtils.setupConstructableRegistryWithConfiguration(configuration);
```

Comment:
Does this interfere with the constructable registry call in the `Hedera` class? I'm surprised this is necessary

Reply:
Shouldn't be interfering with the other call since this sets up some platform class instances which previously had parameterless constructors but now need config param e.g. `VirtualMap`, `VirtualRootNode`, `VirtualNodeCache`.
That's another configuration related change that is initially being addressed in https://github.com/hashgraph/hedera-services/pull/15562.